### PR TITLE
feature: report errors from php cs fixer

### PR DESCRIPTION
**Pull request by caouecs**
## Contribution type

feature

## Description of change

You can choose to get the number of errors from php cs fixer, you don't fix it, and you can choose to have the list of files with problem.

### CODE REVIEW

**Comment by caouecs** [comment]
#74

File: src/Plugin/PhpCsFixer.php [hunk 1]
```diff
@@ -58,8 +63,19 @@ public function __construct(Builder $builder, Build $build, array $options = [])
             $this->args   .= ' --config=' . $builder->interpolate($options['config']);
         }
 
-        if (isset($options['directory']) && $options['directory']) {
-            $this->directory = $builder->interpolate($options['directory']);
+        $this->directory = isset($options['directory'])
+            ? $options['directory']
```
Comment (corpsee) [inline src/Plugin/PhpCsFixer.php]: I guess code should be: `? $builder->interpolate($options['directory'])`.

**Comment by corpsee** [comment]
Thank you @caouecs!

File: src/Plugin/PhpCsFixer.php (hunk 1 above)
Comment (caouecs) [reply src/Plugin/PhpCsFixer.php]: I do the same code than in Phan plugin...

Comment (corpsee) [reply src/Plugin/PhpCsFixer.php]: Values like `%BUILD_PATH%/directory` with [special variables](https://github.com/php-censor/php-censor/blob/master/docs/en/interpolation.md) will not work without interpolation. Phan plugin should also have interpolation but something went wrong :)

**Pull request merged**